CHANGELOG.md

```diff
@@ -17,9 +17,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Schemas refactor: add more Error Checks
 - Schemas refactor: add a json export to method schemas for low-code implementations
 - Schemas refactor: add default value to method types
+- Typings : add AttributeType and ClassAttributeType
+- Blocks : add GetModelAttribute which will replace ModelAttribute in few release
```

Comment:
Please try and keep Changelog alphabetically ordered in order to avoid merge conflicts